tests/small-build/test_built.py

```diff
@@ -0,0 +1,41 @@
+"""
+Test querying ConceptNet for information in the database that results from the
+small test build.
+"""
+
+from conceptnet5.util import get_data_filename
+from conceptnet5.languages import CORE_LANGUAGES, COMMON_LANGUAGES, ALL_LANGUAGES
+from conceptnet5.db.query import AssertionFinder
+
+test_finder = None
+
+
+def setUp():
```

Comment:
Nitpick: `setUp` is I believe the only use of camel case in this document (similarly, in tests/full-build/test_enough_data_exists.py) - is there a reason for it?